### PR TITLE
Minor language changes

### PR DESCRIPTION
**Pull request by himanshujindal1994**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/overview/pdf-electronic-seal-api/quickstarts.md [hunk 1]
```diff
@@ -15,15 +15,15 @@ The client has to call the PDF Electronic Seal API with the appropriate paramete
...
+* **PADES** : This is the latest and improved signature format which is strict, concrete, and secure. For details, see [ETSI TS 102 778-3](https://www.etsi.org/deliver/etsi_ts/102700_102799/10277803/01.02.01_60/ts_10277803v010201p.pdf)  
+* **PKCS7** : This signature format is less stringent than PADES since it permits more PDF changes without invalidating the digital signature. For details, see [ISO 32000-1](https://opensource.adobe.com/dc-acrobat-sdk-docs/standards/pdfstandards/pdf/PDF32000_2008.pdf)
 
 ### TSP Credential Information (*Required*)  
 
 TSP parameters encapsulate the signer's [certificate credential](/overview/pdf-electronic-seal-api/prerequisites/#step-1-procure-certificate-credentials) as well as the associated authentication and authorization data.
 
 * **TSP Name**  (*Required*): Specifies the name of the Trust Service Provider used to generate the certificate.
-* **TSP Credential Id**  (*Required*): Specifies the digital ID stored with the TSP provider that should be used for sealing.
+* **TSP Credential ID**  (*Required*): Specifies the digital ID stored with the TSP provider that should be used for sealing.
```
Comment (piyuverma) [inline src/pages/overview/pdf-electronic-seal-api/quickstarts.md:26]: Specifies the credential ID stored with the TSP provider to be used for sealing.

File: src/pages/overview/pdf-electronic-seal-api/quickstarts.md [hunk 2]
```diff
@@ -40,19 +40,19 @@ The seal field parameters are required to create a new unsigned signature field
...
+  * **Bottom**  (*Required*): The bottom y-coordinate
+  * **Right**  (*Required*): The right x-coordinate
+  * **Top**  (*Required*): The top y-coordinate
 
 The **Location** and **Page Number** fields are **optional** if the signature field is already added in the input PDF document. To add the signature field explicitly, see [how to place a signature field in a PDF](https://www.adobe.com/sign/hub/how-to/add-a-signature-block-to-pdf).
   
 ### Seal Appearance Parameters
 
-Specifies signature field appearance parameters. These are an enum set of display items: NAME, DATE, DISTINGUISHED_NAME, LABELS and SEAL_IMAGE. Specifies the information to display in the signature. NAME and LABELS are the default values.
+Specifies the information to display in the seal. These are an enum set of display items: NAME, DATE, DISTINGUISHED_NAME, LABELS and SEAL_IMAGE. NAME and LABELS are the default values.
```
Comment (piyuverma) [inline src/pages/overview/pdf-electronic-seal-api/quickstarts.md:55]: Specifies the content to be displayed in the seal. These are an enum set of display items: NAME, DATE, DISTINGUISHED_NAME, LABELS and SEAL_IMAGE. NAME and LABELS are the default values.

File: src/pages/overview/pdf-electronic-seal-api/quickstarts.md [hunk 3]
```diff
@@ -40,19 +40,19 @@ The seal field parameters are required to create a new unsigned signature field
 * **Field Name**  (*Required*): The signature field's name. This must be a non-empty string. If signature field with this field name already exist, that field will be used.
```
Comment (piyuverma) [inline src/pages/overview/pdf-electronic-seal-api/quickstarts.md:40]: The seal field parameters are required to create a new signature field or use an existing field.

File: src/pages/overview/pdf-electronic-seal-api/quickstarts.md (hunk 1 above)
Comment (himanshujindal1994) [reply src/pages/overview/pdf-electronic-seal-api/quickstarts.md]: Done

File: src/pages/overview/pdf-electronic-seal-api/quickstarts.md (hunk 2 above)
Comment (himanshujindal1994) [reply src/pages/overview/pdf-electronic-seal-api/quickstarts.md]: Done

File: src/pages/overview/pdf-electronic-seal-api/quickstarts.md (hunk 3 above)
Comment (himanshujindal1994) [reply src/pages/overview/pdf-electronic-seal-api/quickstarts.md:40]: Done

**Pull request merged**